### DISCUSSION TITLE
نحوه تنطیم هیدیفای برای استفاده در سایتهای خارجی (سایتهای ایرانی تونل نشود)

### DISCUSSION THREAD

**Discussion by Ali-Farzin** [hiddify/hiddify-app#559] [Q&A]
سلام
کدام تنظیم موجب می‌شود که آدرس‌های داخل کشور تونل نشود.
به نحوی که با روشن بودن هیدفای، کار روی سایتهای ایرانی حالت معمول را داشته و آدرسهای ایرانی تانل نشود؟

**Comment by LitenBuzzTh** [comment]
در قسمت Settings وارد بخش Routing Assets شده و سه نقطه بالا را زده و Add recommended Assets را زده و سپس geodb و geosite مربوط به ایران را دانلود و انتخاب کنید. در نهایت کانکشن را قطه و مجددا وصل کنید.

**Reply by EKranked** [reply]
چجوری geodb و geosite مربوط به ایران رو بعد دانلود حذف کنیم؟
